src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/cdk/cdk_constructs/adf_codepipeline.py

```diff
@@ -482,3 +487,26 @@ def import_required_arns():
             # pylint: disable=no-value-for-parameter
             _output.append(core.Fn.import_value(arn))
         return _output
+
+
+    def add_pipeline_trigger(self, trigger: Tuple):
```

Comment:
Could you change this to two variables, like `trigger_type` and `trigger_params` or something similar?